tests/bpe.ini

```diff
@@ -85,7 +85,7 @@ encoder=<encoder>
 class=decoders.decoder.Decoder
 name="decoder"
 encoders=[<encoder>]
-embedding_size=9
+embedding_size=10
```

Comment:
Jaký má důvod měnit ty dimenze v těch testech? Většinou jsem se je tam snažím psát tak, aby všechno mělo nějakej jinej rozměr, aby když ten test spadne a mluví o nějakých dimenzích, aby podle toho šlo identifikovat, rychle co je špatně. Tady koukám, že ve velkym nastavuješ rnn size stejně jako embeddingy.

Reply:
Pridal jsem do decoder.Decoder podminku "self.embedding_size != self.output_dimension". Uz uplne nevim, z jakeho duvodu, mozna jako pripravu pro deep RNN decoder. Az ho dodelam, tak to stejne bude jinak